src/lib/component/base/AppLayout/AppLayout.js

```diff
@@ -156,19 +160,20 @@ class AppLayout extends React.PureComponent {
 
     return (
       <Context.Provider value={this.state}>
-        <div className={classes.root}>
+        <div
+          className={classnames(
+            classes.root,
+            !this.props.mobile ? classes.desktop : classes.mobile,
+          )}
+        >
           <div className={classes.topBarContainer}>
-            <ResizeObserver onResize={this.handleTopBarResize} />
+            {/* <ResizeObserver onResize={this.handleTopBarResize} /> */}
```

Comment:
Do we want to remove this? Looks like `topBarHeight` is still being used.

Reply:
Sure